<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDock</class>
 <widget class="QDockWidget" name="LoginDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>175</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_dozent">
       <attribute name="title">
        <string>Dozent</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="email_dozent_lineEdit">
          <property name="placeholderText">
           <string>E-Mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwort_dozent_lineEdit">
          <property name="placeholderText">
           <string>Passwort</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="error_dozent_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="login_dozent_pushButton">
          <property name="text">
           <string>Login als Dozent</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_admin">
       <attribute name="title">
        <string>Admin</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="email_admin_lineEdit">
          <property name="placeholderText">
           <string>Admin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwort_admin_lineEdit">
          <property name="placeholderText">
           <string>Passwort</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="error_admin_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="login_admin_pushButton">
          <property name="text">
           <string>Login als Admin</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
